### DISCUSSION TITLE
Creating a collapse instance shouldn't also toggle the element

### DISCUSSION THREAD

**Discussion by philipstratford** [twbs/bootstrap#38418] [Ideas]
I find the collapse feature in BS5 to be much less pleasant to use than in previous versions. I've just spent quite a lot of time trying to work out why my collapsible elements were immediately toggling when a collapse instance was created. I was doing this:

````
const myCollapse = new bootstrap.Collapse("#myCollapse");
````

expecting to subsequently be able to call

````
myCollapse.show();
````

when I needed to show the content (which is after another element is clicked, not immediately). I couldn't work out why the element was being shown immediately until I eventually realised that that is _by design_, which seems very unintuitive to me. I finally understood that the example in the [docs](https://getbootstrap.com/docs/5.3/components/collapse/#methods):

````
const bsCollapse = new bootstrap.Collapse('#myCollapse', {
  toggle: false
});
````
isn't really a suggestion (i.e. "You can pass the `toggle` option if you want to"), it's a requirement if you don't want the element to immediately be shown.

To me it would be much more intuitive for the basic initialisation with no options parameter to do nothing but return a collapse instance, and require the state to be set explicitly. I.e:

````
const myCollapse = new bootstrap.Collapse("#myCollapse"); // Get the collapse instance
myCollapse.show(); // Show the collapse element
````
or perhaps
````
const myCollapse = new bootstrap.Collapse("#myCollapse").show();
````

At the very least, the documentation should be absolutely explicit that creating a collapse instance will also cause it to be toggled to visible unless an options parameter is passed to prevent it.

**Comment by GeoSot** [comment]
You are absolute right on this 
There is an open issue, waiting to be addressed on v6, as it is a breaking change
https://github.com/twbs/bootstrap/issues/34836
